redisinsight/ui/src/utils/monacoRedisMonarchTokensProvider.ts

```diff
@@ -47,6 +48,13 @@ export const getRedisMonarchTokensProvider = (commands: string[]): monacoEditor.
         ],
         [/[<>=!%&+\-*/|~^]/, 'operator'],
       ],
+      keyword: [
+        [
+          `(${commands.join('|')})\\b`,
+          // '(important|true)\\b',
```

Comment:
do we need this commented-out line of code here?

Reply:
not really, removed